src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -87,6 +87,9 @@ export default App.extend(extend({
     this.attachments.add(model);
 
     Radio.request('ws', 'add', model);
+
+    // to cover the scenario where the AttachmentsView hasn't been shown yet
+    if (this.attachments.length === 1) this.showAttachments();
```

Comment:
The two scenarios I can think of where this will be called:

1. Attachments UI section not visible on initial load of action sidebar. Then a `AttachmentAdded` ws notification is received.
2. Attachments UI section is shown and has one attachment on initial load. Then a `FileRemoved` ws notifciaton is received for the action, clearing the attachments. Then a `AttachmentAdded` ws notification is received.

Reply:
First one is the bug scenario. The second one isn't where the bug happens, since the attachments UI section exists the entire time in that scenario.